### DISCUSSION TITLE
Appreciation post

### DISCUSSION THREAD

**Discussion by Keanegdom** [GitbookIO/community#706] [0. General bugs and questions]
Just a bit of appreciation - I love you for implementing this feature (see the screenshot) 😍 
![Thank you!](https://github.com/user-attachments/assets/fc2aa14b-d43f-494a-99b9-0f2be9b41cbf)
Have a nice day, all of you.

**Comment by Keanegdom** [comment]
By the way - is this feature fully available with the "Plus" plan? Or do I need "Pro"?

**Reply by aikoii** [reply]
Thanks for the GitBook love 💙! This feature is available on our Pro plan
